docs/html/index.html

```diff
@@ -1,23 +1,23 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>Welcome to nginx!</title>
+<title>Welcome to NGINX!</title>
 <style>
 html { color-scheme: light dark; }
 body { width: 35em; margin: 0 auto;
 font-family: Tahoma, Verdana, Arial, sans-serif; }
 </style>
 </head>
 <body>
-<h1>Welcome to nginx!</h1>
-<p>If you see this page, the nginx web server is successfully installed and
-working. Further configuration is required.</p>
+<h1>Welcome to NGINX!</h1>
+<p>If you see this page, NGINX is successfully installed and working.</p>
```

Comment:
My understanding is that "nginx" as software has always been written in lowercase. When we use "NGINX" (in all uppercase), it typically refers to the commercial products. I noticed that in PR #165 you replaced all lowercase instances with uppercase; however, questions about this remained ignored, is this an official rebranding?

Reply:
Please provide the link that confirms that.